HW__2.postman_collection.json

```diff
@@ -0,0 +1,233 @@
+{
+	"info": {
+		"_postman_id": "b531c509-1551-490e-a1a3-d6e2577403e2",
+		"name": "HW_2",
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json",
+		"_exporter_id": "21168131"
+	},
+	"item": [
+		{
+			"name": "HW_2.1",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"exec": [
+							"function excluding_names(names1,names2){\r",
+							"    console.log(_.difference(names1,names2))}\r",
+							" \r",
+							""
+						],
+						"type": "text/javascript"
+					}
+				}
+			],
+			"protocolProfileBehavior": {
+				"disableBodyPruning": true
+			},
+			"request": {
+				"method": "GET",
+				"header": [],
+				"body": {
+					"mode": "raw",
+					"raw": ""
+				},
+				"url": {
+					"raw": "httpbin.org/get",
+					"host": [
+						"httpbin",
+						"org"
+					],
+					"path": [
+						"get"
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "HW_2.4",
+			"event": [
+				{
+					"listen": "prerequest",
+					"script": {
+						"exec": [
+							""
+						],
+						"type": "text/javascript"
+					}
+				},
+				{
+					"listen": "test",
+					"script": {
+						"exec": [
+							"pm.test(\"Status code is 200\", function () {\r",
+							"    pm.response.to.have.status(200);\r",
+							"});\r",
+							""
+						],
+						"type": "text/javascript"
+					}
+				}
+			],
+			"request": {
+				"method": "GET",
+				"header": [],
+				"url": {
+					"raw": "https://rickandmortyapi.com/api/location/{{newID}}",
+					"protocol": "https",
+					"host": [
+						"rickandmortyapi",
+						"com"
+					],
+					"path": [
+						"api",
+						"location",
+						"{{newID}}"
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "post HW_2_2",
+			"event": [
+				{
+					"listen": "prerequest",
+					"script": {
+						"exec": [
+							"const moment = require(\"moment\");\r",
+							"const uid = require(\"uuid\");\r",
+							"pm.environment.set(\"date\", moment().format('MMMM Do YYYY'));\r",
+							"pm.environment.set(\"guid\", uid()); \r",
+							"pm.environment.set(\"randomString\", CryptoJS.SHA256(_.random(1000) + '{{$guid}}').toString()); "
+						],
+						"type": "text/javascript"
+					}
+				},
+				{
+					"listen": "test",
+					"script": {
+						"exec": [
+							"const ajv = require('ajv');\r",
+							"const ajvInstance = new ajv({logger: console});\r",
+							"const jsonObj = pm.response.json();\r",
+							"const schema = {\r",
+							"\r",
+							"            \"date\": \"string\",\r",
+							"            \"guid\": \"string\",\r",
+							"            \"randomString\": \"string\",\r",
+							"}\r",
```

Comment:
It looks like not full schema of our response. 

To prepare required schema we could use the JSON Schema generator: https://www.jsonschema.net/
1. Copy response body
2. Paste to the  JSON Schema generator
3. Generate the schema
4. Beautify the schema
5. Use that schema in our test